### DISCUSSION TITLE
让托盘显示不同的网站图标

### DISCUSSION THREAD

**Discussion by Sunsh4j** [tw93/Pake#564] [Q&A]
如题，在安装之后，桌面会显示不同内容的快捷方式，但是托盘显示的内容都是一样的，这样在打开多个应用的时候，就分不清谁是是的托盘了，所以希望可以让托盘显示不同的网站图标

**Comment by tw93** [comment]
谢谢反馈，这应该是一个Bug，你的事windows的机器吗？我们排查一下这个问题

**Comment by Sunsh4j** [comment]
是的，win10
